src/wcs2_0/pom.xml

```diff
@@ -66,6 +66,11 @@ application directory.
       <artifactId>xmlunit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Looks to be related, but a different issue being addressed?

Reply:
Indeed it's part of its own commit which refers to its own jia ticket: "[GEOS-10156] WCS20GetCoverageResponse does not advertise the list of supported MIME types"

Reply:
My bad - missed the second commit.